charts/scalar-admin-for-kubernetes/templates/_helpers.tpl

```diff
@@ -0,0 +1,116 @@
+{{/* vim: set filetype=mustache: */}}
+{{/*
+Expand the name of the chart.
```

Comment:
This sentence is automatically generated by the `helm create` command. You can see the default sentence in the following way.

```console
$ helm create foo
Creating foo
```

```console
$ ls -l ./foo/templates/_helpers.tpl
-rw-r--r-- 1 ubuntu ubuntu 1742 Jan  9 18:38 ./foo/templates/_helpers.tpl
```
```console
$ sed -n 2p ./foo/templates/_helpers.tpl
Expand the name of the chart.
```

As the above, because it is automatically generated by the Helm, I want to keep this sentence as is.